api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -275,9 +275,6 @@ const (
 	// AroHCP represents the ARO HCP managed service offering
 	AroHCP = "ARO-HCP"
 
-	// RosaHCP represents the ROSA HCP managed service offering
-	RosaHCP = "ROSA-HCP"
```

Comment:
Is this not needed anymore?

Reply:
AFAICT as mentioned in the PR desc: there's no usage nor valid semantic for it atm.
This should have never been added. It does nothing.

Reply:
👍 perfect thanks!